src/manager/BaseMapFishPrintManager.js

```diff
@@ -103,6 +103,34 @@ export class BaseMapFishPrintManager extends Observable {
    */
   customParams = {};
 
+  /**
+   * Custom parameters which can be additionally set on map to determine its
+   * special handling while printing.
+   *
+   * The list of allowed properties is as follows:
+   *  * areaOfInterest
+   *  * bbox
+   *  * center
+   *  * rotation
+   *  * scale
+   *  * useNearestScale
+   *  * dpiSensitiveStyle
+   *  * useAdjustBounds
+   *  * layers
+   *  * width
+   *  * longitudeFirst
+   *  * projection
+   *  * dpi
+   *  * zoomToFeatures
+   *  * height
```

Comment:
A short information about the default properties that will be handled by the manager itself (e.g. the layers) would be a nice addition.

Reply:
Docs are extended in 9464b43